js/templates/account.html

```diff
@@ -1,32 +1,35 @@
 {{#if emailAddress}}
-<div class="mail-account-color" style="background-color: {{accountColor emailAddress}}"></div>
+<div class="mail-account-color app-navigation-entry-bullet" style="background-color: {{accountColor emailAddress}}"></div>
```

Comment:
Shouldn't this be rendered via CSS, similar to the way folders' icons are done?

Reply:
You mean the background color?

Reply:
Not only that, the whole `.mail-account-color` could be moved over to `.mail-account-email:before`. That would make for a cleaner DOM.

Reply:
The bullet is now integrated into server. It's part of the css guidelines.
see https://github.com/nextcloud/server/pull/6399